### ISSUE TITLE
same file in different paths causes false error

### ISSUE THREAD

**Issue by sibelius** [benmosher/eslint-plugin-import#1047]
this is causing no-duplicate lint error

```
import * as searchResultsFilters from './Helpers/SearchResultsFilter';
import { FILTER_FARE, FILTER_ORDERING, FILTER_STOPS } from './Helpers/SearchResultsFilter';
```

**Comment by sibelius** [comment]
this wasn't the problem

this was

```
import * as searchResultsFilters from './Helpers/SearchResultsFilter';
import  from './Helpers/SearchResultsFilter';
```

below does not work
```
 import * as searchResultsFilters, { FILTER_FARE, FILTER_ORDERING, FILTER_STOPS } from './Helpers/SearchResultsFilter';
```

**Comment by ljharb** [comment]
Indeed, `import * as` can't be combined with other things.
